neonvm/tools/vm-builder/files/vminit

```diff
@@ -50,3 +50,11 @@ resize2fs /dev/vda
 # networking
 ip link set up dev lo
 ip link set up dev eth0
+
+# ssh
+/neonvm/bin/ssh-keygen -A
+
+# setup sshd user to support sshd UsePrivilegeSeparation
+mkdir -p /var/empty
+/neonvm/bin/addgroup sshd
+/neonvm/bin/adduser -D -H -G sshd -g 'sshd privsep' -s /neonvm/bin/nologin sshd
```

Comment:
Does this need to run at startup? or can it be built into the image

Reply:
I don't think can be built into the image since the contents of `/etc/group` and `/etc/passwd` depends on the base image that vm-builder uses.

Reply:
Right, but we can run it as part of vm-builder, right? e.g. in `vm-builder/files/Dockerfile.img`, similarly to how we install `vector`

Reply:
We install vector by copying its files into the rootdisk:
https://github.com/neondatabase/autoscaling/blob/a39cf89f7f799143e60f622ef527ced5bb13efc6/neonvm/tools/vm-builder/files/Dockerfile.img#L59-L62

https://github.com/neondatabase/autoscaling/blob/a39cf89f7f799143e60f622ef527ced5bb13efc6/neonvm/tools/vm-builder/files/Dockerfile.img#L84

I don't want to mess up with base image's `/etc/passwd`.

Reply:
I don't quite understand - what's the difference between adding a user in the image vs doing it on startup? is it not the same deterministic change, just made at different times?`

Reply:
Moved user creation to docker build: 1a13ff5e5a60f33ccd694a3aa7a53e0e6deb66bc